router/router.go

```diff
@@ -1505,10 +1521,30 @@ func (rt *HandleT) commitStatusList(responseList *[]jobResponseT) {
 		workspaceID := resp.status.WorkspaceId
 		eventName := gjson.GetBytes(resp.JobT.Parameters, "event_name").String()
 		eventType := gjson.GetBytes(resp.JobT.Parameters, "event_type").String()
-		key := fmt.Sprintf("%s:%s:%s:%s:%s:%s:%s", parameters.SourceID, parameters.DestinationID, parameters.SourceBatchID, resp.status.JobState, resp.status.ErrorCode, eventName, eventType)
+		key := fmt.Sprintf(
+			"%s:%s:%s:%s:%s:%s:%s",
+			parameters.SourceID,
+			parameters.DestinationID,
+			parameters.SourceBatchID,
+			resp.status.JobState,
+			resp.status.ErrorCode,
+			eventName,
+			eventType,
+		)
```

Comment:
[minor] The use of [string.Join](https://pkg.go.dev/strings#Join) might be a good idea here:

```suggestion
		key := strings.Join(
			parameters.SourceID,
			parameters.DestinationID,
			parameters.SourceBatchID,
			resp.status.JobState,
			resp.status.ErrorCode,
			eventName,
			eventType,
			
			":",
		)
```